src/components/Features/index.js

```diff
@@ -32,7 +32,7 @@ const Features = (props) => {
         const maxY = rect.height - element.clientHeight - 50;
         const randomX = Math.floor(Math.random() * maxX);
         // const halfMaxY = maxY / 2;
-        console.log(maxX, maxY, "here");
+        // console.log(maxX, maxY, "here");
```

Comment:
please remove it , do not need to comment out

Reply:
i have removed